server/src/main/java/org/elasticsearch/common/util/concurrent/TaskExecutionTimeTrackingEsThreadPoolExecutor.java

```diff
@@ -89,6 +91,21 @@ public int getCurrentQueueSize() {
         return getQueue().size();
     }
 
+    /**
+     * This returns the percentage of the maximum time spent since the last poll executing tasks
+     */
+    @Override
+    public double getUtilisation() {
```

Comment:
Can we call this `pollUtilization()` to signal more that it is not a read-only method (and start a US/UK/AU spelling fight :slightly_smiling_face:)

Reply:
Addressed all instances of "utilisation" in b9b8f8994b15f768235777d0c272e0a355387bc3